test/test_OpenTURNSFMUPointToFieldFunction.py

```diff
@@ -0,0 +1,100 @@
+    """Return the last simulation times.
+    By default, pyfmi simulation output counts 500 steps.
+    """
+    check_model = pyfmi.load_fmu(path_fmu)
+    check_model.set(["infection_rate", "healing_rate"], input_value)
+    check_result = check_model.simulate()
+    # Get the infected column
+    list_variable = list(check_model.get_model_variables().keys())
+    infected_column = list_variable.index("infected") + 1
+    # +1 stands for time column, inserted as first in the data matrix
+    list_last_infected_value = check_result.data_matrix[infected_column][-5:-1]
+    print(list_last_infected_value)
+    return list_last_infected_value
+
+
+class TestEpid(unittest.TestCase):
+
+    def setUp(self):
+        """Load FMU and setup pure python reference."""
+
+        fmu_name = "epid.fmu"
+        path_example = os.path.dirname(os.path.abspath(
+            otfmi.example.__file__))
+        try:
+            directory_platform = dict_platform[key_platform]
+            self.path_fmu = os.path.join(path_example, "file", "fmu",
+                directory_platform, fmu_name)
+        except KeyError:
+            raise RuntimeError(
+                "Tests are not available on your platform (%s)." % key_platform)
+        except FMUException:
+            raise FMUException("The test FMU is not available on your platform (%s)." % key_platform)
+
+        self.mesh = ot.RegularGrid(2.0, 0.5, 50)
+
+
+    def test_start_time_coherence(self):
+        """Check if incoherent start time raises an error
+        """
+        model_fmu = otfmi.OpenTURNSFMUPointToFieldFunction(
+            self.mesh,
+            self.path_fmu,
+            inputs_fmu=['infection_rate', 'healing_rate'],
+            outputs_fmu=['infected'],
+            start_time=10)
+        self.assertRaises(AssertionError, model_fmu._exec, input_value)
+
+    def test_start_time(self):
+        """Check if start times are taken into account.
+        """
+        model_fmu_1 = otfmi.OpenTURNSFMUPointToFieldFunction(
+            self.mesh,
+            self.path_fmu,
+            inputs_fmu=['infection_rate', 'healing_rate'],
+            outputs_fmu=['infected'],
+            start_time=0)
+        model_fmu_2 = otfmi.OpenTURNSFMUPointToFieldFunction(
+            self.mesh,
+            self.path_fmu,
+            inputs_fmu=['infection_rate', 'healing_rate'],
+            outputs_fmu=['infected'],
+            start_time=1)
+        y1 = model_fmu_1(input_value)
+        y2 = model_fmu_2(input_value)
+        assert(y2[0][0] - y1[0][0] != 0)
+
+    def test_final_time_coherence(self):
+        """Check if incoherent final time raises an error.
+        """
+        model_fmu = otfmi.OpenTURNSFMUPointToFieldFunction(
+            self.mesh,
+            self.path_fmu,
+            inputs_fmu=['infection_rate', 'healing_rate'],
+            outputs_fmu=['infected'],
+            final_time=10)
+        self.assertRaises(AssertionError, model_fmu._exec, input_value)
+
```

Comment:
I had a test that actually tested the value of the function at final time, I see it disappeared

Reply:
Indeed. 
Before, the "epid" model was written in the Python file, then exported as FMU and compared to a manually set value.
Where did this value come from ? I didn't know.

What I changed: the epid.fmu pre-exists among OTFMI examples.
The output value from OTFMI' simulation is compared to the output from PyFMI simulation. As OTFMI interpolation modifies the exact values, the check verifies if OTFMI's output is comprised between PyFMI's 5 last values.